arangod/Pregel/PregelFeature.cpp

```diff
@@ -91,23 +90,22 @@ std::pair<Result, uint64_t> PregelFeature::startExecution(
 
   // check the access rights to collections
   ExecContext const& exec = ExecContext::current();
-  if (!exec.isSuperuser()) {
```

Comment:
If the user is a superuser then the user should have access to the collection. Therefore removing this short-cut as Superuser is not well-defined.